OpenRA.Mods.Common/Traits/Capturable.cs

```diff
@@ -25,9 +25,9 @@ public class CapturableInfo : ITraitInfo
 		public readonly int CaptureThreshold = 50;
 		public readonly bool CancelActivity = false;
 
-		public object Create(ActorInitializer init) { return new Capturable(this); }
+		public override object Create(ActorInitializer init) { return new Capturable(this); }
 
-		public bool CanBeTargetedBy(Actor captor, Player owner)
+		public bool FrozenCanBeTargetedBy(Actor captor, Player owner)
```

Comment:
There is nothing specific to `FrozenActor`s in this method, so please leave the name as it was.